bytes/bytes_test.ts

```diff
@@ -0,0 +1,34 @@
+import {
+  bytesFindIndex,
+  bytesFindLastIndex,
+  bytesEqual,
+  bytesHasPrefix
+} from "./bytes.ts";
+import { assertEqual, runTests, test } from "./deps.ts";
+
+test(function testFindIndex() {
+  const i = bytesFindIndex(
+    new Uint8Array([1, 2, 0, 1, 2, 0, 1, 2, 0, 1, 3]),
+    new Uint8Array([0, 1, 2])
+  );
+  assertEqual(i, 2);
+});
+test(function testFindLastIndex() {
```

Comment:
please have blank line between test cases. 

Don't prefix test cases with "test" but use "bytes" ... EG "bytesFindLastIndex"

Reply:
OK, but is there prettier rule for that ?